src/dbnode/storage/index.go

```diff
@@ -1897,6 +1924,26 @@ func newNamespaceIndexMetrics(
 			"loaded-docs-per-query",
 			tally.MustMakeExponentialValueBuckets(10, 2, 16),
 		),
+		queryExhaustiveSuccess: scope.Tagged(map[string]string{
+			"exhaustive": "true",
+			"result":     "success",
+		}).Counter("query"),
+		queryExhaustiveInternalError: scope.Tagged(map[string]string{
+			"exhaustive": "true",
+			"result":     "error_internal",
+		}).Counter("query"),
+		queryNonExhaustiveSuccess: scope.Tagged(map[string]string{
+			"exhaustive": "false",
+			"result":     "success",
+		}).Counter("query"),
+		queryNonExhaustiveInternalError: scope.Tagged(map[string]string{
+			"exhaustive": "false",
+			"result":     "error_internal",
+		}).Counter("query"),
+		queryNonExhaustiveLimitError: scope.Tagged(map[string]string{
+			"exhaustive": "false",
+			"result":     "error_require_exhaustive",
```

Comment:
Can these have the full status rather than an `exhaustive` and a `result` field?